### ISSUE TITLE
TensorBoard 5. Hyperparam: timestamp 6:06: say "lower learning rate ...will end up with lower cost"

### ISSUE THREAD

**Issue by e7dud7e** [udacity/dlnd-issue-reports#231]
In TensorBoard lesson 5: "Choosing Hyperparameters", at time stamp 6:06, Matt says "Having a lower **cost**, it will take longer to run, but will also end up with lower cost".  I think he means to say "Having a lower **learning rate**... will end up with lower cost."
